src/components/Sidebar/Sidebar.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { useContext } from 'react';
+import { PostDetails } from './PostDetails/PostDetails';
+import { ActivePostContext } from '../../utils/Store';
+import classNames from 'classnames';
```

Comment:
Consider using the `classnames` library for defining classes conditionally, which you are already doing here. This is a good practice as per the checklist.